content/blog/oauth2-for-hpe-greenlake-data-services-cloud-console.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Implementing OAuth 2 Flow for Data Services Cloud Console's Client
+  Application "
+date: 2021-10-14T12:46:19.488Z
+author: Ron Dharma
+authorimage: /img/face-portrait-small.jpg
+thumbnailimage: /img/dscc-icon-transparent.png
+tags:
+  - data-services-cloud-console
+---
+## HPE GreenLake API Security
+
+The major principle for public (Northbound) API to HPE GreenLake and Data Services Cloud Console is *secured interface that support authentication and authorization of the resource owner.* To support the previously mentioned principle, the HPE GreenLake implements the Oauth2 authorization granting the resource owner authorization rights without embedding the resource owner's user credential as in the API streams. This goal is achieved using the Oauth2 authorization and REST API, where every API call will embed the access token as part of the HTTP REST API URL stream using the keyword: *bearer*.
```

Comment:
Please see my comments on the previous post - some of those will apply here as well.

Reply:
I decided to remove this paragraph and create much shorter paragraph.